packages/web-app/hooks/useTime.tsx

```diff
@@ -0,0 +1,21 @@
+import { useState, useEffect } from "react";
+import { useBlockNumber } from "wagmi";
+
+const BLOCK_MINING_TIME = 12; // seconds
+
+export const secondsBetweenBlocks = (currentBlock: number, targetBlock: number) => {
+	return (targetBlock - currentBlock) * BLOCK_MINING_TIME;
+};
+
+export const useTimeToBlock = (targetBlock: number) => {
```

Comment:
If we use the block.timestamp instead of the block number we can avoid block mining time calculations (~12 sec per block) and any misalignment by block time differs.
I'm here assuming we are saving the block, could we save the timestamp?

Reply:
It's a general good practice to use the block number instead of the block timestamp in the smart contracts. But I wouldn't oppose to use the timestamp instead of the block number for the disputes.